corgi/core/migrations/0098_load_missing_container_errata_builds.py

```diff
@@ -0,0 +1,63 @@
+from collections import defaultdict
+
+from django.db import migrations
+from django.db.models import Count
+
+from corgi.collectors.errata_tool import ErrataTool
+from corgi.tasks.errata_tool import slow_load_errata as current_slow_load_errata
+from corgi.tasks.prod_defs import brew_tag_to_et_prefixes
+from corgi.tasks.tagging import NO_MANIFEST_TAG
+
+
+def load_missing_container_errata_build(apps, schema_editor):
+    ProductStream = apps.get_model("core", "ProductStream")
+    CollectorErrataProductVersion = apps.get_model("collectors", "CollectorErrataProductVersion")
+
+    variants_by_stream = defaultdict(set)
+    # Allow this migration to be re-run in case of timeout or other failure
+    # Only get the streams we haven't processed yet
+    for stream in (
+        ProductStream.objects.annotate(num_components=Count("components"))
+        .filter(num_components__gt=0)
+        .exclude(tags__name=NO_MANIFEST_TAG)
+        .exclude(meta_attr__has_key="container_errata_checked")
+    ):
+        if stream.brew_tags:
+            for brew_tag in stream.brew_tags.keys():
```

Comment:
Minor, but I think you can just do this in one line, since brew_tags defaults to an empty list:
```suggestion
        for brew_tag in stream.brew_tags:
```

Reply:
Oh yeah, thanks. Anyway, I moved this to the exclude part of the query instead.